airflow/providers/databricks/hooks/databricks.py

```diff
@@ -69,6 +69,14 @@
 
 SPARK_VERSIONS_ENDPOINT = ("GET", "api/2.0/clusters/spark-versions")
 
+CREATE_REPO_ENDPOINT = ("POST", "api/2.0/repos")
+
+DELETE_REPO_METHOD = "DELETE"
+DELETE_REPO_ROUTE = "api/2.0/repos/{repo_id}"
+
+UPDATE_REPO_METHOD = "PATCH"
+UPDATE_REPO_ROUTE = "api/2.0/repos/{repo_id}"
```

Comment:
If we're trying to maintain consistency, would something like the follow make more sense?

```suggestion
# with a TPL postfix maybe?
DELETE_REPO_ENDPOINT = ("DELETE", "api/2.0/repos/{repo_id}")

UPDATE_REPO_ENDPOINT = ("PATCH", "api/2.0/repos/{repo_id}")
```